src/FerryTempo.js

```diff
@@ -90,8 +94,27 @@ export default {
           boatDelay = epochTimeStamp - epochScheduledDeparture;
         }
 
+        // Calculate the BoatETA as a function of the progress we have made on the journey. We use this value unless we do not get
```

Comment:
Nit: format multi-line comments with `/**`
```suggestion
        /** Calculate the BoatETA as a function of the progress we have made on the journey. We use this value unless we do not get
        *   a departureTime value from WSDOT for the boat, in wich case we will use the arrivalTimeEta computed from the Eta field and now().
        **/
```